src/test/e2e/api/openapi/openapi.e2e.test.ts

```diff
@@ -50,6 +51,24 @@ test('the generated OpenAPI spec is valid', async () => {
         await SwaggerParser.validate(body);
     } catch (err) {
         console.error(err);
-        return false;
+        throw err;
```

Comment:
Just want to get this clear in my own head - this suggests that errors will now bubble when previously they wouldn't. This doesn't mean that old behaviour in our APIs will now error if it doesn't like the shape of the data, right?

Reply:
Yeah, so this is only for tests that we perform. I'm not sure why I wrote `return false` previously. I might've been under the impression that it would fail the test, but I had to cause an exception later on. In this case, though, I should probably swap it with `expect(true).toBe(false)` or something.

But to be clear: this is _not_ runtime code. This is test code. If it breaks, then that means we've discovered errors we didn't know we had, right?